tex/gregoriotex-syllable.tex

```diff
@@ -1091,13 +1091,17 @@
       \GreNoBreak%
     \fi%
     \ifgre@eolshiftsenabled%
-      %this shift will shorten the line somewhat to allow the lyrics to extend under the custos should TeX decide to insert a line break after the syllable being ended
-      \gre@calculate@eolshift{\dimexpr(\gre@dimen@enddifference-\gre@skip@bar@lastskip)\relax }%
-      \gre@debugmsg{barspacing}{End of line shift: \the\gre@dimen@eolshift}%
-      \ifdim\gre@dimen@eolshift<\gre@skip@bar@lastskip %
-        \gre@dimen@eolshift=\gre@skip@bar@lastskip\relax %
-      \fi %
-      \kern\glueexpr(-\gre@dimen@eolshift)\relax%
+      \ifdim\wd\gre@box@syllablenotes>0pt\relax%
```

Comment:
It's happening because the last "element" in the score is a texverb rather than a real element, and the code only looks one ahead.  What is the correct behavior?

1. Put `\GreLastOfScore` on the last real element, or
2. Put `\GreLastOfScore` on the last "thing" of the score, regardless of what it is, or
3. Something more complex than that (please explain).